### PR TITLE
hr - croatian

### PR DESCRIPTION
**Pull request by kenante**
New language. HR

### CODE REVIEW

**Comment by hivokas** [comment]
Placeholders (e.g. `:attribute`) don't need to be translated.

**Comment by kenante** [comment]
Fixed

File: resources/lang/hr.json [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+    "Detach": "Odspoji",
+    "Detach Selected": "Odspoji Selektovano",
+    "Delete Selected": "Izbriši Selektovano",
+    "Force Delete Selected": "Nasilno Izbriši Selektovano",
+    "Restore Selected": "Vrati Selektovano",
+    "Restore Resource": "Vrati Resurs",
+    "Restore": "Vrati",
+    "Force Delete Resource": "Nasilno Izbriši Resurs",
+    "Force Delete": "Nasilno Izbriši",
+    "The :resource was created!": "Resurs je kreiran!",
```
Comment (hivokas) [inline resources/lang/hr.json]: Seems like it's not fixed.

File: resources/lang/hr.json [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+    "Are you sure you want to delete this resource?": "Da li ste sigurni da želite izbisati resurs?",
+    "Are you sure you want to delete the selected resources?": "Da li ste sigurni da želite izbisati selektovani resurs?",
+    "Are you sure you want to detach this resource?": "Da li ste sigurni da želite odspojiti resurs?",
+    "Are you sure you want to detach the selected resources?": "Da li ste sigurni da želite odspojiti selektovani resurs?",
+    "Are you sure you want to force delete this resource?": "Da li ste sigurni da želite nasilno izbisati resurs?",
+    "Are you sure you want to force delete the selected resources?": "Da li ste sigurni da želite nasilno izbisati selektovani resurs?",
+    "Are you sure you want to restore this resource?": "Da li ste sigurni da želite vratiti resurs?",
+    "Are you sure you want to restore the selected resources?": "Da li ste sigurni da želite vratiti selektovani resurs?",
+    "No :resource matched the given criteria.": "Ne: Resurs ne odgovara zadanom kriteriju",
```
Comment (hivokas) [inline resources/lang/hr.json]: Seems like it's not fixed.

**Comment by hivokas** [comment]
Please, fix the placeholders.

**Pull request closed without merging**